extensions/wikia/DesignSystem/services/templates/DesignSystemGlobalNavigationOnSiteNotifications_index.php

```diff
@@ -0,0 +1,19 @@
+<div id="onSiteNotificationsEntryPoint"
+     class="wds-global-navigation__notifications-menu wds-global-navigation__notifications-menu wds-dropdown notifications-container">
```

Comment:
👍  for the notificaitons-menu
The notificaitons-container is related to bubbles.